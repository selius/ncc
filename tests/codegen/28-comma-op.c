int main()
{
	int a, b, c;
	float d, e, f;

	a = 5;
	b = 6;

	c = (a += b, b - a);
	__print_int(c);

	d = 4.7;
	e = 9.2;
	
	f = (e -= d, e - d);
	__print_float(f);

	return 0;
}
