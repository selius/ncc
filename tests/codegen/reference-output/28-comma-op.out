-5
-0.200000
